### ISSUE TITLE
Global websites are no longer redirecting to the right sites

### ISSUE THREAD

**Issue by itsmingjie** [hackclub/global#77]
Tested `es.hackclub.com` and `japan.hackclub.com`. Neither are correctly redirecting to the respective sites anymore.

**Comment by jakjakob** [comment]
btw hackclub.es isn't working

**Comment by Oasx** [comment]
true

**Comment by grymmy** [comment]
Yeah - this is bad - I will prioritize investigating this tomorrow.

**Comment by LucasHT22** [comment]
Looks like it's because they aren't in https://github.com/hackclub/dns/blob/main/hackclub.com.yaml

**Comment by Monoverde888** [comment]
i think the hackclub.es domain expired

**Comment by toonoeichi** [comment]
also thai that now had moved to th and it now not working.

Original : thai.hackclub.com (https://github.com/hackclub/dns/blob/f26cd20f89d2f81b9f07df601b0cccee4570bc9c/hackclub.com.yaml#L3219)
New : th.hackclub.com (doesn't exist on hackclub.com.yaml in dns!)
